src/commands/misc/asciify.js

```diff
@@ -13,8 +13,8 @@ module.exports = class Asciify extends Command {
     )
   }
 
-  run ({ channel }, text) {
-    const bigtext = figlet.textSync(text, {
+  run ({ channel, message }) {
+    const bigtext = figlet.textSync(message.cleanContent.split(' ').slice(1).join(' '), {

```

Comment:
Splitting by 1 can still cause problems on servers that use custom prefixes that contain spaces. I suggest making a "CleanStringParameter"

Reply:
Sounds good

Reply:
Fixed by #629